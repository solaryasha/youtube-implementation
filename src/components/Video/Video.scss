.video-container {
    position: relative;
    width: 100%;

    &:before {
        content: '';
        display: block;
        padding-top: 56.25%;
    }
    .video {

        position: absolute;
        bottom: 0;
        top: 0;
        left: 0;
        right: 0;


        .video-player {
            min-height: 480px;
            width: 100%;
            height: 100%;
        }
       
    }
    
}