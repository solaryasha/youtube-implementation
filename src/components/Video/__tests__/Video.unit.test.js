import React from 'react';
import {shallow} from 'enzyme';
import Video from '../Video';

describe('Video ',()=>{
    test('renders video component correctly', () => {
        const wrapper = shallow(
            <Video id='vQzqTTaWefY' />
        );

        expect(wrapper).toMatchSnapshot();
    })
    test('renders video component correctly', () => {
        const wrapper = shallow(
            <Video id='vQzqTTaWefY' />
        );

        expect(wrapper).toMatchSnapshot();
    })
    
})