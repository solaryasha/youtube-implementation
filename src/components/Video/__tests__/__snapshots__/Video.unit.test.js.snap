// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video  renders video component correctly 1`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="video-player"
      frameBorder="0"
      src="https://www.youtube.com/embed/vQzqTTaWefY"
      title="video"
    />
  </div>
</div>
`;

exports[`Video  renders video component correctly 2`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="video-player"
      frameBorder="0"
      src="https://www.youtube.com/embed/vQzqTTaWefY"
      title="video"
    />
  </div>
</div>
`;
